pkg/services/sqlstore/search_builder.go

```diff
@@ -135,7 +146,7 @@ func (sb *SearchBuilder) buildTagQuery() {
 	// this ends the inner select (tag filtered part)
 	sb.sql.WriteString(`
 		GROUP BY dashboard.id HAVING COUNT(dashboard.id) >= ?
-		ORDER BY dashboard.id` + dialect.Limit(int64(sb.limit)) + `) as ids
+		ORDER BY dashboard.id as ids
```

Comment:
Hmm wonder why we had limit in the inner select statement before?

Reply:
yea, did not understand it either, and I wrote it

Reply:
think for performance reasons